From: Venkat Duvvuru <venkatkumar.duvvuru@avagotech.com>
Date: Wed, 30 Dec 2015 01:29:02 -0500
Subject: [PATCH 08/28] be2net: remove unused error variables
Patch-mainline: v4.5-rc1
Git-commit: 04e888de2d72b587f81a18e42cb9af2c020ef63b
References: bsc#963844 FATE#320192

eeh_error, fw_timeout, hw_error variables in the be_adapter structure are
not used anymore. An earlier patch that introduced adapter->err_flags to
store this information missed removing these variables.

Signed-off-by: Venkat Duvvuru <venkatkumar.duvvuru@avagotech.com>
Signed-off-by: Sathya Perla <sathya.perla@avagotech.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/emulex/benet/be.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/ethernet/emulex/benet/be.h
+++ b/drivers/net/ethernet/emulex/benet/be.h
@@ -547,10 +547,6 @@ struct be_adapter {
 
 	u32 beacon_state;	/* for set_phys_id */
 
-	bool eeh_error;
-	bool fw_timeout;
-	bool hw_error;
-
 	u32 port_num;
 	char port_name;
 	u8 mc_type;
